docs/blog/2018-08-09-swag-store/index.md

```diff
@@ -36,7 +36,7 @@ Here’s the short version:
 3.  Request a discount code with your name and email
 4.  Use that discount code at checkout to get one free item from the swag store!
 
-> **NOTE:** The way this works is by checking that your GitHub account has at least one merged PR in the [Gatsby organization][org]. However, **contributing code is _not_ the only way to contribute to open source!** Check out [the “how to contribute” docs][contribute] for more examples of contributions that qualify for free swag. Email us at <mailto:team@gatsbyjs.com> if you’re a non-code contributor and want your free swag.
+> **NOTE:** The way this works is by checking that your GitHub account has at least one merged PR in the [Gatsby organization][org]. However, **contributing code is _not_ the only way to contribute to open source!** Check out [the “how to contribute” docs](/docs/how-to-contribute/) for more examples of contributions that qualify for free swag. Email us at <mailto:team@gatsbyjs.com> if you’re a non-code contributor and want your free swag.
```

Comment:
This does look to be fixing an error, but can you use our reference style links instead?

e.g. at the bottom of the file add

```markdown
[contribute]: /docs/how-to-contribute/
```

Reply:
Did I understand correctly - I have to delete the link I was editing and add it to the bottom of the file:
 `[contribute]: /docs/how-to-contribute/`
?

Reply:
Exactly; yep! So it would look like it did with `[the “how to contribute” docs][contribute]` and then just add the reference style link below! Make sense?

Reply:
@DSchau like this?